packages/firestore/src/remote/datastore.ts

```diff
@@ -247,8 +247,10 @@ export async function invokeRunAggregationQueryRpc(
     queryToTarget(aggregateQuery.query._query)
   );
 
-  //while invoking streaming RPC, grpc_connection requires parent in the request
-  //while rest_connection excludes it
+  /**
+   * Include the parent in the request for grpc_connection.
```

Comment:
I think at this point you can simply remove the comment. It is just repeating what the code below is doing. Generally, you only use inline comments to explain the _why_ of the code, not the _what_.